docs/azure_ad_config_guide.rst

```diff
@@ -138,3 +138,92 @@ Add this path to your projects ``urls.py`` file.
         path('oauth2/', include('django_auth_adfs.urls')),
         ...
     ] 
+    
+Step 3 - Register and configure an Azure AD frontend application
+--------------------------------------------------------
+Just like we did with our backend application in step 1 We have to register a new app for our frontend. Here we need to set it to SPA and the redirect URI value must match with the domain where your frontend application is located(eg. http://localhost:3000).
```

Comment:
I think it would be nice to specify that this is only required to use the Django Rest Framework token authentication through a SPA.